source/reference/connection-string.txt

```diff
@@ -51,6 +51,8 @@ The components of this string are:
        specific database using these credentials after connecting to
        the :program:`mongod` instance.
 
+       If username or password contains an at-sign ("@") or colon (":") then the username and/or password must be URL encoded. At-sign ("@") is replaced with "%40" (3 characters), while colon (":") is replaced with "%3A" (3 characters).
+
```

Comment:
Do you mind if we go more generic and then more specific example?

If the username or password contains reserved URI characters, you must escape the characters per RFC 2396. For example, if your username is s@m, you must escape the @ character when specifying the username in the connection string; i.e. s%40m.